package.json

```diff
@@ -22,7 +23,9 @@
   ],
   "dependencies": {
     "axios": "^0.21.0",
+    "browserify": "^17.0.0",
```

Comment:
This should only a dev dependency, right?

Reply:
If your building an npm you are correct.